core/domain/skill_services.py

```diff
@@ -1147,3 +1147,87 @@ def filter_skills_by_mastery(user_id, skill_ids):
         if skill_id in filtered_skill_ids:
             arranged_filtered_skill_ids.append(skill_id)
     return arranged_filtered_skill_ids
+
+
+def get_untriaged_skill_summaries(
+    skill_summaries, skill_ids_assigned_to_some_topic, merged_skill_ids):
+    """Returns a list of skill summary dicts for all skills that are
+    untriaged.
+
+    Args:
+        skill_summaries: list(SkillSummary). The list of all skill summary
+            domain objects.
+        skill_ids_assigned_to_some_topic: set(str). The set of skill ids which
+            are assigned to some topic.
+        merged_skill_ids: list(str). List of skill IDs of merged skills.
+
+    Returns:
+        list(dict). A list of skill summary dicts for all skills that
```

Comment:
Don't pass dicts around except at app boundaries ... I think I mentioned this before. Always use domain objects.

Domain objects may have a to_dict() method that you can invoke just before passing the data through an app boundary.

Reply:
done.